### DISCUSSION TITLE
openrdf.org

### DISCUSSION THREAD

**Discussion by riannella** [eclipse-rdf4j/rdf4j#4381] [General]
Hi, just noting that the openrdf.org domain name is no longer registered (ie any one could acquire it).
It is used for the vocab in the REST API: https://rdf4j.org/documentation/reference/rest-api/

**Comment by abrokenjester** [comment]
Yes, that's a leftover, really: the openrdf.org domain was owned by the now defunct Sesame project (of which RDF4J is the successor). In order to keep compatibility with older systems we initially didn't update any of these vocabulary URLs, and we haven't really made it a priority since to switch over.

It's a fair point that we really should start addressing this, and introduce new vocabulary using the rdf4j.org domain.

**Reply by abrokenjester** [reply]
https://github.com/eclipse/rdf4j/issues/4401
